ai-platform-unified/notebooks/official/automl/automl-text-classification.ipynb

```diff
@@ -0,0 +1,1413 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    output_folder_name = [your-batch-prediction-output-folder]
</pre><p> </p><p>Ditto. You can just use a real name and append timestamp if it needs to be globally unique.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/360/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Reply:
<p>Ack. See paragraph that precedes this cell. The customer doesn't control the folder name.</p>